### ISSUE TITLE
Tags are not changing when browsing the site with Next JS 13

### ISSUE THREAD

**Issue by BWBama85** [garmeeh/next-seo#1103]
**Describe the bug**
I have just started using Next SEO so hopefully I haven't just missed something completely obvious.

I Next SEO setup the following way:

next-seo.config.ts

```
// next-seo.config.ts
import type { NextSeoProps } from 'next-seo';

export const NEXT_SEO_DEFAULT: NextSeoProps = {
  title: 'The Wilson Net built by Brent Wilson',
  description:
    'The Wilson Net is a place for Brent Wilson to showcase his life.',
  openGraph: {
    type: 'website',
    locale: 'en_IE',
    url: 'https://www.thewilsonnet.com',
    title: 'The Wilson Net built by Brent Wilson',
    description:
      'The Wilson Net is a place for Brent Wilson to showcase his life.',
    images: [
      {
        url: 'https://www.test.ie/og-image-a-01.jpg',
        width: 800,
        height: 600,
        alt: 'Og Image Alt A',
        type: 'image/jpeg',
        secureUrl: 'https://www.test.ie/secure-og-image-a-01.jpg',
      },
    ],
    siteName: 'The Wilson Net',
  },
  twitter: {
    handle: '@brentwilson85',
    cardType: 'summary_large_image',
  },
};
```

layout.tsx
```
      <head>
        <meta charSet="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <link rel="icon" href="/favicon.ico" />
        <NextSeo
          useAppDir={true}
          titleTemplate="%s"
        />
      </head>
```
head.tsx for root (homepage)
```
import { NextSeo } from 'next-seo';
import { NEXT_SEO_DEFAULT } from './next-seo.config';
import type { NextSeoProps } from 'next-seo';

export default function Head() {
  const updateMeta: NextSeoProps = {
    ...NEXT_SEO_DEFAULT,
    title: 'The Wilson Net by Brent Wilson',
    description: 'The Wilson Net is a place for Brent Wilson to showcase his life.',
    titleTemplate: '%s',
  };
  return (
    <NextSeo {...updateMeta} useAppDir={true} />
  )
}
```

head.tsx for /resume
```
import { NextSeo } from 'next-seo';
import { NEXT_SEO_DEFAULT } from '../../next-seo.config';
import type { NextSeoProps } from 'next-seo';

export default function Head() {
  const updateMeta: NextSeoProps = {
    ...NEXT_SEO_DEFAULT,
    title: 'Richard Brent Wilson\'s Resume',
    description: 'A detailed resume for Richard Brent Wilson',
    titleTemplate: '%s',
  };
  return (
    <NextSeo {...updateMeta} useAppDir={true} />
  )
}
```

**Reproduction**
The site is live here and you can click around and see that the title tag and meta desc does not change.

https://thewilsonnet.com/

**Expected behavior**
The title and desc should update to reflect what page you are on.

**Additional context**

**Comment by BWBama85** [comment]
I'm sorry, I missed the very first line in the app readme. Somehow...

**Comment by BWBama85** [comment]
This is fixed in Next 13.0.4 if you want to update the docs.
